ndcube/tests/test_ndcubesequence.py

```diff
@@ -1,264 +0,0 @@
-import datetime
```

Comment:
Why is this file deleted?

Reply:
Is the idea that when I port the new fixture-based sequence tests, they'll slot in here rather than still remaining deleted?  Maybe best to not delete this file as part of this PR and simply have it fixed by another PR?